### PR TITLE
Update emailjs

### PR DESCRIPTION
**Pull request by Guy-deMessimy**

### CODE REVIEW

**Review by AmjadLille — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
+.env.sample
```
Comment (LoicBrassart) [inline .gitignore]: If you add the `.env.sample` file here, it will be totally useless

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove the .env.asample file from .gitignore

**Review by AmjadLille — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**